implementations/rust/ockam/ockam_entity/src/credential/util.rs

```diff
@@ -75,7 +75,7 @@ where
             let _l = if let Some(l) = s.size_hint() { l } else { 0 };
             let mut buf = Vec::new();
             while let Some(a) = s.next_element()? {
-                #[cfg(all(feature = "no_std", not(feature = "std")))]
+                #[cfg(all(not(feature = "std"), not(feature = "std")))]
```

Comment:
That looks strange to me. Could we put actual push operation outside of cfg attribute, and only handle error inside attribute scope?

Reply:
Heh, busted. Over-zealous search/replace! 

I didn't write this but I'm assuming this was a test for a `heapless::vec::Vec` overflow. I also don't think you can have an attribute inside an expression. 

I'm just going to remove the duplicate cfg check.

Reply:
I don't think you can have attribute inside an expression, but could that look something like this?
```
let r = buf.push(a);
#[cfg(all(feature = "no_std", not(feature = "std")))]
{
    r.map_err(|_| DError::invalid_length(_l, &self))?;
}
```

Reply:
*doh* - that makes sense!